From: David Disseldorp <ddiss@suse.de>
Date: Thu, 29 Oct 2015 18:33:14 +0100
Subject: [PATCH] target/pr: fix core_scsi3_pr_seq_non_holder() caller
Patch-mainline: Not yet, SES2 clustered LIO/RBD
References: bnc#952666

The incorrectly placed semi-colon means that RESERVATION CONFLICT is
returned to persistent reservation non-holders, regardless of the
core_scsi3_pr_seq_non_holder() return value.

Signed-off-by: David Disseldorp <ddiss@suse.de>
---
 drivers/target/target_core_pr.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/target/target_core_pr.c
+++ b/drivers/target/target_core_pr.c
@@ -629,7 +629,7 @@ check_nonholder:
 
 	if (core_scsi3_pr_seq_non_holder(cmd, pr_reg_type,
 					 sess->se_node_acl->initiatorname,
-					 registered_nexus));
+					 registered_nexus))
 		return TCM_RESERVATION_CONFLICT;
 	return 0;
 }
